measure.rb

```diff
@@ -3593,6 +3659,7 @@ def self.set_surface_interior(model, spaces, surface, surface_id, interior_adjac
       surface.setSpace(create_or_get_space(model, spaces, Constants.SpaceTypeUnconditionedBasement))
     elsif ["basement - conditioned"].include? interior_adjacent_to
       surface.setSpace(create_or_get_space(model, spaces, Constants.SpaceTypeLiving))
+      @cond_bsmnt_surfaces << surface
```

Comment:
We should also do this in the `set_surface_exterior` method.

Interzonal surfaces (e.g., a surface between two zones) doesn't really have a convention for what's "interior" vs "exterior". So, a FoundationWall in HPXML that is between conditioned basement and garage could be described as interior=garage and exterior=conditioned basement.

Reply:
Good point. I didn't notice it's putting surface to adjacent zone. Thanks.